std/math/exponential.d

```diff
@@ -112,44 +113,22 @@ if (isFloatingPoint!(F) && isIntegral!(G))
 
     enum uint bias = F.max_exp - 1;
 
-    static if (is(F == float))
```

Comment:
Issue 1: qualifiers ignored - slow generic branch taken for all non-mutable FP types.